.circleci/src/jobs/build_administration.yml

```diff
@@ -7,31 +7,22 @@ steps:
       path: ~/project
   - restore_cache:
       keys:
-        - v1-node-modules-{{ checksum "package.json" }}-{{checksum "package-lock.json" }}
+        - v2-node-modules-{{ checksum "package.json" }}-{{checksum "package-lock.json" }}
   - install_dart_linux
   - install_protobuf_linux
   - run:
-      name: Install node dependencies
-      command: "[ ! -d node_modules ] && npm ci --loglevel warn --yes || echo package.json and package-lock.json unchanged. Using cache."
+      name: Install node dependencies on root level
+      command: "cd .. && [ ! -d node_modules ] && npm ci --loglevel warn --yes || echo package.json and package-lock.json unchanged. Using cache."
```

Comment:
needed for npm workspaces

Reply:
because the post install would lead to an error as you send me the suggestion with ignore scripts? and we execute the post install later manually

Reply:
yes, but you missed the `--ignore-scripts` tag here, when calling the command

Reply:
sorry i just didn't see the `not`